README.md

```diff
@@ -79,24 +79,33 @@ This application written in C does:
      cd ../..
      ```
   Note: You may have to re-plug you dongle if it was already connected, or play with `udev` if not automatically detected.
-  
+
   5. Clone this repository:
      ```bash
      git clone https://github.com/Guenael/rtlsdr-wsprd
      ```
-  
+
   6. Build the application:
      ```bash
      cd rtlsdr-wsprd
      make
      sudo make install
      ```
-  
+
   7. Finally, start the application with the right parameters/options for you (frequency, callsign, locator etc... Fake example below):
      ```bash
      rtlsdr_wsprd -f 2m -c A1XYZ -l AB12cd -g 29
      ```
 
+## Container Image
+
+As an alterative to the above steps, a pre-built container image containing rtlsdr-wsprd is available for use with [Docker](https://www.docker.com/) or [Podman](https://podman.io/).
```

Comment:
`rtlsdr-wsprd` in a docker container, nice! Thanks for this :)
And some new stuff to try on my side, I didn't know podman.io!